lib/Default.js

```diff
@@ -0,0 +1,170 @@
+'use strict'
+
+const ms = require('pretty-ms')
+
+const { colorize } = require('./Util')
+const CONST = require('./Constants')
+const format = require('./Format')
+
+module.exports = () => ({
+  print () {
+    return Object.keys(this.types).map(type =>
```

Comment:
fat arrow avoid return

Reply:
Same here, access to `this` makes it impossible to use arrows...